createFilename.m

```diff
@@ -89,7 +89,7 @@
 [~, ~, ~] = mkdir(expParameters.outputDir);
 [~, ~, ~] = mkdir(fullfile(expParameters.outputDir, modality));
 
-if cfg.eyeTracker
+if isfield(cfg,'eyeTracker')
```

Comment:
The logic of using a simple `if cfg.eyeTracke` boolean (0/1) is because if I have the ET in my design and I run its code at my home, I will encounter problems because the actual ET hardware is not connected. So, if I am at my home or I don't use the ET in my design, in `setParameters.m` I will put 

``` matlab
cfg.eyeTracker    = false;      % Set to 'true' if you are testing in MRI and want to record ET data
```

Reply:
+1

Reply:
question: if I want to send a or to this suggesting the quick and dirty solution, I should send the pr to Ceren's master, am I right? Will Remi will see it?

Reply:
not sure...

Reply:
You would have to send it to Ceren's master indeed. But github has now this thing that allows maintainers to push directly to a PR without opening a PR themselves. See the allow "edits bys maintainers" tick box: if Ceren has ticked it then you could just checkout the branch that she has pushed from (by double clicking on it in git kraken once you added her repo as a remote), commit some stuff and push.

![Screenshot 2020-05-26 19:19:22](https://user-images.githubusercontent.com/6961185/82930541-dae30800-9f85-11ea-9669-83340cb57b92.png)

Reply:
Otherwise I would only see the end results when Ceren merges your PR onto her PR.

Reply:
But as I have all your repos as remotes I can "see" everything you do on gitkraken as long as you push it. :-)

#EvenOnGitHubICanBeCreepy

Reply:
looking for linking a pr outside a repo but no success at the moment, anyway [here it is](https://github.com/CerenB/CPP_BIDS/pull/1)

Reply:
Ok I merged into my master. but not sure what's the rest is about... looking for Marco's review.

Reply:
ok, closed that pr and pushed the modification here, it is quick and dirty. LMK if and how you would improve it